### ISSUE TITLE
[BUG] App won't get past quick setup

### ISSUE THREAD

**Issue by bako3** [laurencedawson/sync-for-reddit#1219]
**Describe the bug**
Hi! I'm having issues opening up Sync Pro. I'm on version v22.5.4 and I just switched to an S22 Ultra. This version works fine on my S21 Ultra but I can't seem to get past the quick setup on my S22 Ultra. After either skipping or going through the quick setup, the app will crash and give me a notification that says "Sync is having trouble starting. Please redownload directly from Google Play". After this crash, trying to launch the app won't give me the quick setup anymore and will just proceed to crash and give me the same notification again.

- I've tried clearing my cache and data for the app which allows me to go through the quick setup again, but it will once again crash.
- I've also tried uninstalling the app and reinstalling. Same issue
- Leaving the beta and redownloading. Same issue.
- I have downloaded the Sync Free app (v19.0.20) and that seems to startup fine.

**To Reproduce**
1. Open App
2. Go through Quick Setup (either skip or go all the way through)
3. App Crashes
4. Opening app again is immediate crash

**Smartphone (please complete the following information):**
 - Device: Samsung S22 Ultra
 - OS: Android 12, One UI 4.1, AVCK
 - App: Sync Pro
 - Version: v22.5.4

**Additional context**
Reddit thread: https://www.reddit.com/r/redditsync/comments/ujz8ud/crash_after_quick_setup/

**Comment by laurencedawson** [comment]
Did you install from Google play?

**Comment by bako3** [comment]
> Did you install from Google play?

Yes I did

**Comment by laurencedawson** [comment]
Did you modify the APK or move it to the secure folder on your device?

**Comment by bako3** [comment]
I did have a version of it in the Secure Folder. I did not remove it from the Secure Folder when I uninstalled it from my regular apps. When I tried to reinstall it from Google Play Store, it seemed to have just copied it from the Secure Folder rather than downloading it again. I uninstalled from Secure Folder as well and it downloaded from Google Play and I was able to launch it fine! I guess it wasn't an issue with Sync, but with Samsung. Thank you for helping me troubleshoot!
